LeezeIt/LeezeIt/Controller/SignUpViewController.swift

```diff
@@ -7,7 +7,7 @@
 
 import UIKit
 
-class CreateAccountViewController: UIViewController, UITextFieldDelegate {
+class SignUpViewController: UIViewController, UITextFieldDelegate {
```

Comment:
move the UITextFieldDelegate in extension

Reply:
Resolved